src/utils/hasMethod.spec.js

```diff
@@ -0,0 +1,10 @@
+const { expect } = require('chai');
+const { hasMethod} = require("./index");
+
+describe('Utils - hasMethod', function suite() {
+  it('should correctly handle method detection', function () {
```

Comment:
Split into different cases

Reply:
Agree, will add it along https://github.com/dashevo/wallet-lib/pull/295#discussion_r701729118